src/coreclr/jit/llvm_types.h

```diff
@@ -0,0 +1,83 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/*
+ * Types used in LLVM compilation that are abstractions of the VM types
+*/
+
+#ifndef _LLVM_TYPES_H_
+#define _LLVM_TYPES_H_
+
+#include "corinfo.h"
+
+struct TypeDescriptor
+{
+    size_t                fieldCount;
+    CORINFO_FIELD_HANDLE* fields;
+};
+
+struct FieldDesc
+{
+private:
+    int                  _fieldOffset;
+    CORINFO_CLASS_HANDLE _classHandle;
+    CorInfoType          _corType;
```

Comment:
Nit: the Jit coding convention is to have fields prefixed with `m_`: `m_fieldOffset`, etc.

Reply:
ok, I copied GenTree https://github.com/dotnet/runtimelab/blob/ea61843f3125bba9e12c7844f70b0406883e6179/src/coreclr/jit/gentree.h#L857 but will change it.

Reply:
As you may have noticed, the actual Jit code does not match up with the conventions :). But it's nice for at least new code to be uniform.